OrbitCore/PathTest.cpp

```diff
@@ -35,3 +35,79 @@ TEST(Path, FileExistsDevNull) {
   EXPECT_TRUE(Path::FileExists(filename));
 }
 #endif
+
+TEST(Path, JoinPathPartsEmpty) {
+  std::string actual = Path::JoinPathParts({});
+  std::string expected{};
+  EXPECT_EQ(actual, expected);
+}
+
+TEST(Path, JoinPathPartsRelativeSlash) {
+  std::string actual =
+      Path::JoinPathParts({"dir1/dir2", "dir3/dir4/", "dir5/dir6"});
+  std::string expected;
```

Comment:
nit: move declaration inside ifdef here and below

Reply:
Done